spec/system/jobseekers/jobseekers_can_search_for_jobs_spec.rb

```diff
@@ -418,14 +415,6 @@
 
       expect_page_to_show_jobs([senior_leader, it_support])
       expect_page_not_to_show_jobs([job1, job2, job3, job4, maths_job1, maths_job2, headteacher, deputy_head, teaching_assistant, sendco, pastoral, other])
-
-      uncheck "IT support"
```

Comment:
We have already tested checking/unchecking/combining filters so I think this final step is not necessary